blackjack/actions.py

```diff
@@ -23,14 +24,16 @@ def stick_or_twist(hand: Hand, deck: Deck) -> None:
     Raises:
         Exception: Raises an exception if the user does not input a valid input of "s" or "t"
     """
-    move = input("Stick or twist? Input s or t")
     while True:
+        move = input("Stick or twist? Input s or t: ")
+
         if move == "t":
             twist(hand, deck)
         elif move == "s":
             print("Player has chosen to stick.")
+            game_config.playing = False
         else:
-            raise Exception("Please enter either s or t")
+            print("Invalid input. Please enter either s or t.")
```

Comment:
I Feel like you have some duplicated code here, in asking for user input, would it be better to move that into a seperate function?

Reply:
Have moved a part of this into a separate function